elliott/tests/test_verify_kernel_tag_cli.py

```
@@ -0,0 +1,471 @@
+    def test_no_stop_ship_tag(self):
+        config = [{"packages": ["foo"], "integration_tag": "bar"}]
+        packages, tag = get_kernel_packages_and_tag(config)
+        self.assertEqual(packages, set())
+        self.assertEqual(tag, "")
+
+    def test_multiple_entries(self):
+        config = [
+            {"packages": ["kernel"], "stop_ship_tag": "tag1", "integration_tag": "ic1"},
+            {"packages": ["kernel-rt"], "stop_ship_tag": "tag2", "integration_tag": "ic2"},
+        ]
+        packages, tag = get_kernel_packages_and_tag(config)
```

Ruff (0.16.1)

[warning] Unpacked variable `tag` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)